lti_consumer/models.py

```diff
@@ -288,7 +340,7 @@ def _generate_lti_1p3_keys_if_missing(self):
             )
 
         # Doesn't do anything if model didn't change
-        self.save()
+        return self.save()
```

Comment:
Is this necessary if the model didn´t change?

Reply:
I just added it to always have an explicit return, let me check if there is no linting issues, if not I will remove it.

Reply:
Not adding the return rases issues with pylint R1710 rule.